modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,276 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from modin.experimental.sklearn.model_selection.train_test_split import train_test_split
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score, mean_squared_error
+
+import multiprocessing as mp
+import ray
+
+
+ray.init(log_to_driver=False)
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "modin_type_y",
```

Comment:
Let's add this parameter to all the tests.

Reply:
This added for all tests